src/styles/components/page-header/styles.less

```diff
@@ -1,71 +1,102 @@
-& when (@page-header-enabled) {
```

Comment:
I actually renamed the `src/styles/layout/page-header/styles.less` file and deleted the old `src/styles/components/page-header/styles.less` file, but I forgot to make a commit at that stage :sob: